src/app/home/panel/cards/dashboard/loaders/dashboard-main-info-loader/dashboard-main-info-loader.component.scss

```diff
@@ -0,0 +1,32 @@
+.loader {
+  height: 1.5rem;
+  width: 10rem;
+  margin: .5rem 0
+}
+
+.loader-amount-title {
+  height: .75rem;
+  width: 9rem;
+  margin: 1.25rem 0 0 .25rem;
+}
+
+.gradient {
+  border-radius: 1rem;
+  animation-name: loaderShimmer;
+  animation-fill-mode: forwards;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
+  background: linear-gradient(to right, #eeeeee 8%, #ffffff 18%, #ececec 33%);
+  background-size: 50rem 7rem;
+  animation-duration: 1.2s;
+}
+
+
+@keyframes loaderShimmer {
+  0% {
+    background-position: -30rem 0
+  }
+  100% {
+    background-position: 30rem 0
+  }
+}
```

Comment:
Those styles are almost the same as in 'budgets-list-loader'. Try to make it more reusable.